packages/gestalt/src/Link.css

```diff
@@ -9,15 +9,17 @@
   text-decoration: underline;
 }
 
-.inlineBlock {
-  composes: inlineBlock from "./Layout.css";
+.inlineFlex {
```

Comment:
It's unexpected to get styles for alignItems and justifyContent from a class called "inlineFlex" — by our typical usage, this would only set the `display` style. Also, from the logic in InternalLink, it looks like we'll get itemsCenter and justifyCenter regardless. So it would be more obvious to add both those explicitly in InternalLink and keep this class focused…or actually not even defining this class (or `flex` below), and just use the existing classes directly from Layout.css.

Reply:
DONE

Reply:
DONE